ClemBot.Bot/bot/cogs/info_cog.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import discord
+import discord.ext.commands as commands
+import bot.extensions as ext
+from bot.consts import Colors, Claims
+from bot.messaging.events import Events
+
+log = logging.getLogger(__name__)
+
+class InfoCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @ext.command()
+    @ext.required_claims(Claims.moderation_infraction_view)
+    @ext.ignore_claims_pre_invoke()
+    async def info(self, ctx, user: discord.User = None):
+        #If the command is invoked without a specified user, it will return info on the calling user
+        if not user:
+            user = ctx.author
+        
+        log.info(f'User {ctx.author} has ran info command on user {user.name}')
+
+        embed = discord.Embed(title = f'Guild Member Information', color=Colors.ClemsonOrange)
+        embed.set_author(name=f'{user.name}#{user.discriminator}', icon_url=user.avatar_url)
```

Comment:
`str(user) == f'{user.name}#{user.discriminator}'`
```suggestion
        embed.set_author(name=str(user), icon_url=user.avatar_url)
```